lib/services/sign_in.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_sign_in/google_sign_in.dart';
+
+class SignIn {
+  static GoogleSignIn googleSignIn = GoogleSignIn(
+    scopes: <String>[
+      'email',
+      'https://www.googleapis.com/auth/contacts.readonly',
+    ],
+  );
+  static GoogleSignInAccount currentUser;
+
+  static Future<void> handleSignIn() async {
+    try {
+      await SignIn.googleSignIn.signIn();
+    } catch (error) {
+      print(error);
+    }
+  }
```

Comment:
this is a redundant function (you should just catch an error wherever you attempt sign in so it can properly presented to user)